src/gui/optionsdialog.ui

```diff
@@ -2456,7 +2456,7 @@
                   <bool>false</bool>
                  </property>
                  <property name="alignment">
-                  <set>Qt::AlignCenter</set>
+                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
```

Comment:
if you only want left alignment then just remove the whole `<property name="alignment>` section.
i.e. remove the following:
```xml
<property name="alignment">
  <set>Qt::AlignCenter</set>
</property>
```

Reply:
Done.